code/modules/uplink/uplink_items/nukeops.dm

```diff
@@ -658,12 +658,13 @@
 	purchasable_from = UPLINK_NUKE_OPS | UPLINK_CLOWN_OPS
 
 /datum/uplink_item/device_tools/medkit
-	name = "Syndicate Combat Medic Kit"
-	desc = "This first aid kit is a suspicious brown and red. Included is a combat stimulant injector \
-			for rapid healing, a medical night vision HUD for quick identification of injured personnel, \
-			and other supplies helpful for a field medic."
+	name = "Syndicate Combat Medical Suite"
+	desc = "This first aid kit is a suspicious brown and red. Included is an unloaded combat chemical injector \
```

Comment:
this shit aint brown and red for like 4 years

Reply:
![image](https://user-images.githubusercontent.com/23585223/200123316-3c916740-ab01-4d4a-a8d7-7d33da697580.png)

Reply:
Fuck